libpod/options.go

```diff
@@ -1456,6 +1456,10 @@ func WithNamedVolumes(volumes []*ContainerNamedVolume) CtrCreateOption {
 				return errors.Wrapf(err, "error processing options for named volume %q mounted at %q", vol.Name, vol.Dest)
 			}
 
+			if err = parse.ValidateVolumeCtrDir(vol.Dest); err != nil {
+				return errors.Wrapf(err, "error validating destination path for named volume %q mounted at %q", vol.Name, vol.Dest)
```

Comment:
This looks like you created a stutter.  Does vol.Dest get mentioned in the err twice?
Don't start error messages with the word "error" otherwise they are printed on error with

Error: error ...

I know there are other errors in options.go with this problem, please fix them while you are at it.

I would like to eventually go through the error messages in all of podman and remove any that begin with "error".

Also see what this error message looks like when it happens.  Is this information the user needs or does it hide the error amongst a flood of words.

Reply:
Yes this adds stutter i verified let me fix this :)

Reply:
@rhatdan resolved for `options.go` i'll create a seperate PR cleaning this in entire project.